src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -278,14 +278,17 @@ def k8s_install_cli(client_version='latest', install_location=None):
 
     file_url = ''
     system = platform.system()
-    base_url = 'https://storage.googleapis.com/kubernetes-release/release/{}/bin/{}/amd64/{}'
+    base_url = 'https://storage.googleapis.com/kubernetes-release/release/v{}/bin/{}/{}/{}'
```

Comment:
Think there's a bug here.
If `client_version` is `latest`, `https://storage.googleapis.com/kubernetes-release/release/stable.txt` returns a version that includes the `v` prefix so then `base_url` would have `/release/vv1.6.3`?